### PR TITLE
further enhances logging

### LINKED ISSUE #496: set combinedError graph calls better
Rather than log the combinedError being set as context it needs to be setExtra.

### PR DESCRIPTION
**Pull request by csell5**
part 2 of #496 but setting better.

closes #496

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { inspect } from '@xstate/inspect';
 import * as Sentry from '@sentry/browser';
 import { Integrations } from '@sentry/tracing';
 
-import { debug } from './config';
+import config, { debug } from './config';
```
Comment (brettski) [inline src/main.js:6]: This smells odd

**Review by brettski — APPROVED** [review]
+ 1 comment

File: src/main.js (hunk 1 above)
Comment (csell5) [reply src/main.js:6]: it's the default export. prolly could be named better though.

Comment (brettski) [reply src/main.js:6]: yes, that was the smell

**Pull request merged**